<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrailStation</class>
 <widget class="QMainWindow" name="TrailStation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrailStation</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuitApp"/>
   </widget>
   <widget class="QMenu" name="menuAccaunt">
    <property name="title">
     <string>Аккаунт</string>
    </property>
    <addaction name="actionEnterToAccount"/>
    <addaction name="actionQuitAccount"/>
   </widget>
   <widget class="QMenu" name="menuAboutProgramm">
    <property name="title">
     <string>О программе</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuAccaunt"/>
   <addaction name="menuAboutProgramm"/>
  </widget>
  <action name="actionAddAdmin">
   <property name="text">
    <string>Администратор</string>
   </property>
  </action>
  <action name="actionAddCashier">
   <property name="text">
    <string>Кассир</string>
   </property>
  </action>
  <action name="actionAddPassanger">
   <property name="text">
    <string>Пассажир</string>
   </property>
  </action>
  <action name="actionAccountAdmin">
   <property name="text">
    <string>Администратор</string>
   </property>
  </action>
  <action name="actionAccountCashier">
   <property name="text">
    <string>Кассир</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Пассажир</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionQuitApp">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionEnterToAccount">
   <property name="text">
    <string>Войти</string>
   </property>
  </action>
  <action name="actionQuitAccount">
   <property name="text">
    <string>Выйти</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
